### PR TITLE
Kapacitor 1.5.9 release notes

### PR DESCRIPTION
**Pull request by kelseiv**
Closes https://github.com/influxdata/docs-v2/issues/2331

Do we want to add more contextual info about how the 1.5.8 release was pulled in the changelog?, and/or add contextual info to the release notes on which feature/functionality were originally in 1.5.8 and then rolled back into 1.5.9 in the change log/release notes?

Including @veereshhosakere...recall support and edge teams working with us on verbiage when we pulled down 1.5.8.

### CODE REVIEW

File: content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md [hunk 1]
```diff
@@ -6,6 +6,22 @@ menu:
     parent: About the project
 ---
 
+## v1.5.9 [TBD]
```
Comment (kelseiv) [inline content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md:9]: Do we need to add any of the content that we pulled for the 1.5.8 release back into the 1.5./9 release?

File: content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md [hunk 2]
```diff
@@ -6,6 +6,22 @@ menu:
...
+
+## Features
+
+- Send full event payload on pagerduty resolve, thanks @asvinours!
+- Add default color theme to teams alerts, thanks @NoamShaish!
```
Comment (kelseiv) [inline content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md:14]: Are these Microsoft Teams alerts...or something else?

Comment (docmerlin) [reply content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md:14]: @kelseiv  Yes, microsoft teams alerts.

**Review by timhallinflux — APPROVED** [review]

File: content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md [hunk 3]
```diff
@@ -38,11 +30,26 @@ If you’ve installed this release, please roll back to v1.5.7 as soon as possib
...
+- Add the default color theme to [Microsoft Teams](/kapacitor/v1.5/event_handlers/microsoftteams/) alerts, thanks @NoamShaish!
+- Add barrier handling to [FlattenNode](/kapacitor/v1.5/nodes/flatten_node/) to ensure points are successfully emitted.
+
 ### Bug fixes
 
+- Ensure large batch writes with `influx` gzip are completely written to InfluxDB.
+- Update function node name `ServiceNow` handler to camelcase.
+- Fix memory leaks in `JoinNode` and `UnionNode`.
+- Avoid infinite hang when closing Kafka writer and prevent the timeout error that occurred when updating the Kafka configuration file (`kapacitor.conf`) via http.
+- Remove support for `darwin/386` builds (Go no longer supports), and add support for `darwin/arm64` builds.
```
Comment (docmerlin) [inline content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md:45]: We don't support darwin/arm64

**Review by docmerlin — CHANGES_REQUESTED** [review]

**Review by docmerlin — APPROVED** [review]

**Pull request merged**